app/javascript/utils/DelegationRequests.js

```diff
@@ -18,12 +19,14 @@ import { post } from "../fetch/requester";
 export async function delegateQuestions(answers, buildingId, email, firstName, lastName, addAnswers) {
   let delegations = [];
   for (const answer of Object.values(answers)) {
+    console.log("EMAIL: - " + email);
```

Comment:
Oops haha these were the things I put in yesterday while we were debugging but please remove all the print statements in your PR!